dev-app/android/build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
     ext {
         buildToolsVersion = "31.0.0"
         minSdkVersion = 21
-        compileSdkVersion = 31
+        compileSdkVersion = 33
```

Comment:
all looks good to me but not sure if any concerns with bumps here so will leave for others to take a look too (but the cocoapods bump all looks fine 👍 )